### ISSUE TITLE
kdb list documentation outdated

### ISSUE THREAD

**Issue by Chemin1** [ElektraInitiative/libelektra#3122]
Our website mentions the `kdb list` command [here](https://www.libelektra.org/manpages/kdb-list). However, the command is not found in my Docker container. Is the documentation outdated or did I miss something when setting up my container?

```
jenkins@44fb32900e01:~/workspace/elektra-build-docker$ kdb list -v
The command kdb list is not known
Usage: kdb <command> [args]

kdb is a program to manage Elektra's key database.
Please run a command with -H or --help as args to show a help text for
a specific command.

Known commands are:
cache	Enable, disable or clear the cache.
cmerge	Three-way merge of KeySets.
complete	Show suggestions how to complete key names.
convert	Convert configuration.
cp	Copy keys within the key database.
editor	Use your editor for editing KDB.
export	Export configuration from the key database.
file	Prints the file where a key is located.
find	Find keys with a given regex.
gen	Execute a code-generator template.
get	Get the value of an individual key.
global-mount	Globally mount given plugins.
global-umount	Unmounts a global plugin from key database.
gmount	Globally mount given plugins.
gumount	Unmounts a global plugin from key database.
help	View the man page of a tool
import	Import configuration to the key database.
list-commands	List available kdb commands.
list-tools	List all external tools
ls	List the names of keys below a given name.
merge	Three-way merge of KeySets.
meta-get	Get a metavalue.
meta-ls	Get all meta information of an individual key.
meta-rm	Remove a metakey.
meta-set	Set a metavalue.
meta-show	Print all metakeys along with their value for the given key.
mount	Mount a new backend.
mv	Move configuration within the key database.
plugin-check	Do some basic checks on a plugin.
plugin-info	Print information about a plugin.
plugin-list	List all available or provided plugins.
remount	Remount an existing backend with a different filename.
rm	Remove key(s) from key database.
set	Set the value of an individual key.
sget	Get the value of an individual key within a shell script.
shell	Start a kdb shell.
smount	Mount a new backend by specification. All necessary plugins will be loaded automatically, depending on the metadata.
spec-mount	Mount a new backend by specification. All necessary plugins will be loaded automatically, depending on the metadata.
test	Run key database test suite.
umount	Unmount backend from key database.
vset	Set a value together with a validation regex.
```

**Comment by sanssecours** [comment]
>  Is the documentation outdated or did I miss something when setting up my container?

As far as I know the website was not properly updated in quite a long time (see also: issue #2897).

**Comment by Chemin1** [comment]
Thank you for the hint! I now checked the repo more thoroughly and could not find `kdb list -v`, so I assume this issue will be resolved as soon as #2897 is resolved.

**Comment by markus2330** [comment]
The command is now called `kdb plugin-list`.

http://a7.libelektra.org/ should you already give up-to-date information.
